cmd/s3-peer-client.go

```diff
@@ -19,125 +19,74 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
-	"net/rpc"
 	"net/url"
 	"path"
 	"sync"
-	"time"
 )
 
 type s3Peers struct {
-	// A map of peer server address (in `host:port` format) to RPC
-	// client connections.
-	rpcClients map[string]*AuthRPCClient
+	// A map of peer server address (in `host:port` format) to
+	// BucketMetaState interface
+	bmsClients map[string]BucketMetaState
 
-	mutex *sync.RWMutex
-
-	// Is single-node?
-	isDistXL bool
+	// Is distributed?
+	IsDistXL bool
 
 	// Slice of all peer addresses (in `host:port` format).
-	peers []string
+	Peers []string
```

Comment:
@fwessels Yes, Peers does include "itself". I was thinking of peers (as in servers with identical roles) as the group of servers in the cluster, and not with respect to the current server, so the name didn't seem misleading to me. Let me know if changing it to "Nodes" or "AllNodes" or something else would help readability, and I can do it.